server/api/rounds.py

```diff
@@ -709,6 +709,55 @@ def sample_batches(
 def validate_round(round: dict, election: Election):
     validate(round, CREATE_ROUND_REQUEST_SCHEMA)
 
+    total_batches = 0
```

Comment:
@jonahkagan Using `targeted_contest_ids` is simplifying some things, but now in trying to access the contest object itself is being odd. I tried 

```for contest_id in targeted_contest_ids:
            single_contest = election.contests[election.contests.index(contest_id)]```

But I got the error `error: Value of type "Iterable[Contest]" is not indexable` so that evidently doesn't work. What would you suggest?

Reply:
ah, the problem here might be that `election.contests` is not actually a list, it's an iterator (a lazy list returning items one by one, in this case, from the database). so to fix that error, i think doing `list(election.contests)` will fix it

that being said, it might be easier to change `targeted_contest_ids` to `targeted_contests` and make it a list of contest objects instead of ids, that way you don't need to go searching for the contest objects again

```python
targeted_contests = [contest for contest in election.contests if contest.is_targeted]
if set(round["sampleSizes"].keys()) != {c.id for c in targeted_contests}:
  ... etc ...
```